tools/src/wyvern/target/corewyvernIL/type/StructuralType.java

```diff
@@ -101,7 +101,7 @@ public boolean isSubtypeOf(ValueType t, TypeContext ctx) {
 		if (t instanceof NominalType) {
 			StructuralType st = ((NominalType) t).getStructuralType(ctx, null);
 			if (st == null) 
-				return false; // abstract type; I am not a subtype of this
+				return true; // abstract type; I am always a subtype of this
```

Comment:
Does the test case fail if this is reverted?  It seems wrong for a random structural type to be a subtype of a nominal type that is abstract.